core/src/main/java/com/google/instrumentation/stats/Stats.java

```diff
@@ -26,7 +26,7 @@
    * Returns the default {@link StatsContextFactory}.
    */
   public static StatsContextFactory getStatsContextFactory() {
-    return statsManager.getStatsContextFactory();
+    return statsManager == null ? null : statsManager.getStatsContextFactory();
```

Comment:
@sebright We have made our code to skip reporting logic if `getStatsContextFactory()` returns null, so we don't need the no-op impl any more. I am going to close that issue.